release notes/v0.37.0.md

```diff
@@ -0,0 +1,73 @@
+k6 v0.37.0 is here! 🎉 Mainly it contains fixes and ongoing efforts with refactoring.
+
+## New Features!
+
+### Added an option to output k6 logs to a file through --log-output ([#2285](https://github.com/grafana/k6/pull/2285))
+
+This is on top of the supported stdout/stderr and loki instance.
+
+The feature speaks for itself with examples of usage:
+
+```sh
+k6 run --log-output file=./k6.log --logformat json ./examples/stages.js
+```
+
+with a defined minimal log level:
+
+```sh
+k6 run --log-output file=./k6.log,level=info --logformat json ./examples/stages.js
+```
+
+Thanks, @[alyakimenko](https://github.com/alyakimenko) for the contribution!
+
+**Docs**: [Using file output](https://k6.io/docs/using-k6/options/#file)
+
+### Added experimental basic event loops ([#882](https://github.com/grafana/k6/issues/882))
+
+We added basic event loop functionality ([#2228](https://github.com/grafana/k6/pull/2228) and [#2373](https://github.com/grafana/k6/pull/2373)). This is a first step and currently isn't used by anything, but is now available to extensions. In following releases this will be used by k6 and will mean that some APIs will start having callbacks or return Promises. But first it's likely that it will be used in some experimental extensions.
+
+If you are an extension developer please use it and give your feedback. But take into consideration that it's likely that the API may change
+
+## Extensions
+
+`v0.37.0` finalizes ([#2376](https://github.com/grafana/k6/pull/2376) and [#2384](https://github.com/grafana/k6/pull/2384)) the switching of our internal modules to [a new Go/JavaScript module API](https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension). As the last internal the GRPC module was refactored ([#2365](https://github.com/grafana/k6/pull/2365)).
+
+It's important to highlight that the old API is deprecated and will be removed in the next k6's release (`v0.38.0`). Every extension that isn't using that API will get a warning message, like:
+
+```sh
+WARN[0000] Module 'k6/x/sql' is using deprecated APIs that will be removed in k6 v0.38.0, for more details on how to update it see https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension
```

Comment:
Is a single "minor" revision too soon to remove API?

Reply:
It's not the single "minor," at least once we already mentioned deprecation/removal in our release notes (previous release) and [the link](https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension) declares that the actual announcement of the new API was in October 2021 so it seems reasonable to remove, no? :thinking: 

@codebien maybe you have something to add here?

Reply:
The new API has been around for a bunch of versions actually and we have at least since the last version told users to use it instead of the old one.

While we don't want to break any user scripts we have more or less made no such promises for internal k6 APIs and this is more or less what we are breaking here. For the record we will likely completely break more or less anything doing with metrics at some point as well. Which in practice means all extensions. And while in general we do not try to break extensions on purpose, a lot of them touch clearly very internal APIs that should not have been exported probably to begin with. 


So for me "attracting developers" should come with a warning of "we will break the API to make everything better so beware"